docs/troubleshooting/migrate_cstor_pools.md

```diff
@@ -0,0 +1,229 @@
+# Pool Migration when node and underlying disks were lost
+
+## Intro
+- There can be a situations where you lost node and disks attached to the node were also lost. This is very common for users having Kubernetes autoscalar feature enabled and nodes will be scaledown and scaleup based on the demand.
+
+- When the new node come up, then disks that are attached to the node were also new(It doesn't have any previous data).
+
+Consider an example where you have following nodes in a Kubernetes cluster with disks attached it:
+
+    Worker-1 (bd-1(w1), bd-2(w1) are attached to worker1)
+
+    Worker-2 (bd-1(w2), bd-2(w2) are attached to worker 2)
+
+    Worker-3 (bd-1(w3), bd-2(w3) are attached to worker 3)
+
+**NOTE**: Disks attached to a node are represented by a blockdevice(bd) in OpenEBS installed cluster. A block device is of the form `bd-<some-hash>`. For example, bd-1(w1), bd-3(w2) etc are BD resources. For illustration purpose hash is not included in BD name, e.g. `bd-1(w1)` represents a block device attached to worker-1.
+
+## What happens if Node and underlying disk lost?
+If node and underlying disk attached to the node are lost in your kubernetes cluster then cStor pool manager pods will be in pending state and the cStor pools that are running on lost node and volumes in that pool will be marked offline. Workloads using those cStor volumes will not be able to perform read and write operations on the volume if volume is not configured with HA configuration(i.e storage replica count is 3).
+
+## How can this be fixed?
+We can perform few manual [steps](#steps-to-be-followed-to-move-the-pool-from-failed-node-to-new-node) to recover from this situtation. But before we do this, the tutorial will illustrate a node and underlying disk lost situtations. So essentially we are trying to do the following:
+
+**NOTE**: If CStorVolumes are **Healthy** then only perform below steps are recommended.
+
+**__Migrate CStorPool when nodes and underlying disks were lost__**
+
+## Reproduce the Node and underlying disk lost situtation?
+
+#### In cloud environment
+    Able to remove the nodes by deleting the nodes from the K8s cluster managed by autoscalar group(ASG). If external volumes/disks are attached to the node with an option deleting the nodes should delete the disks attached to the node. Currently it was experimented in EKS, GCP managed kuberntes cluster.
```

Comment:
We can remove nodes from the Kubernetes cluster managed by an auto-scaler group(ASG) in cloud-managed Kubernetes services e.g. EKS and GCP. If the nodes are removed then the attached volumes/disks on the removed node will also get deleted if the attached volumes/disks are ephemeral.